charts/rancher-turtles/templates/pre-install-job.yaml

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
Could we have a value that can be used to enable or disable this? Perhaps in the values.yaml we can have something like this:

```yaml
features:
  cleanup-rancher-webhooks: true
```

I can then add to the values.yaml in this section to enable/delete the labels update (#178)